packages/tuono/src/build/config/create-json-config.spec.ts

```diff
@@ -20,6 +20,20 @@ describe('createJsonConfig', () => {
     )
   })
 
+  it('should process config with only server property including origin', async () => {
+    const sampleConfigWithOptional = {
+      server: { host: 'h', origin: 'o', port: 1 },
+    }
+
+    await createJsonConfig(sampleConfigWithOptional)
+
+    expect(writeFileSpy).toHaveBeenCalledWith(
+      expect.any(String),
+      expect.stringContaining(JSON.stringify(sampleConfig)),
```

Comment:
This test should be failing, but it is not.
You are comparing  `sampleConfigWithOptional` with `sampleConfig`

```suggestion
      expect.stringContaining(JSON.stringify(sampleConfigWithOptional)),
```

The reason is that we weren't clearing the `writeFileSpy`. 
I'm going to fix this on `main`.

Reply:
I added the mock reset on `main`. 
Could you please merge those changes and update the code as above?

Reply:
Hey @pveierland, 
if you don't mind, we can take over and resolve the conflicts on our end

Reply:
Sorry, I missed the latest sub-comment here. Happy if you guys take over :+1: